pkg/graveler/ref/repository_iterator_test.go

```diff
@@ -25,7 +25,7 @@ func TestRepositoryIterator(t *testing.T) {
 	}
 
 	t.Run("listing all repos", func(t *testing.T) {
-		iter := ref.NewRepositoryIterator(context.Background(), db, 3)
+		iter := ref.NewDBRepositoryIterator(context.Background(), db, 3)
```

Comment:
why is the kv repository iterator not tested?

Reply:
Yicks.
Done - thanks

Reply:
And if we can also reuse the same test code for the db iterator 💪